src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/AssessmentsController.java

```diff
@@ -86,6 +88,9 @@ public Integer createUpdateRequirement(final @PathVariable("assessment-id") Inte
     var principal = getPrincipalFromJwt(authentication);
     log.info("createUpdateRequirement invoked on behalf of principal: {}", principal);
 
+    if (requirement.getWeighting() == null) {
+      throw new ValidationException(ERR_EMPTY_BODY);
```

Comment:
I think we should be able to add the necessary validation to the API spec, then apply `@Valid` annotation - will approve for now but that is the preferred approach.